test/framework/executor_itf/executor_itf.cpp

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "framework/executor_itf/executor_itf.hpp"
+
+#include "ametsuchi/impl/postgres_command_executor.hpp"
+#include "ametsuchi/impl/postgres_query_executor.hpp"
+#include "ametsuchi/tx_executor.hpp"
+#include "backend/protobuf/proto_permission_to_string.hpp"
+#include "framework/config_helper.hpp"
+#include "framework/test_logger.hpp"
+#include "interfaces/permissions.hpp"
+#include "logger/logger.hpp"
+#include "logger/logger_manager.hpp"
+#include "module/shared_model/mock_objects_factories/mock_command_factory.hpp"
+#include "module/shared_model/mock_objects_factories/mock_query_factory.hpp"
+
+using namespace iroha::integration_framework;
+using namespace iroha::ametsuchi;
+using namespace common_constants;
+
+namespace {
+  logger::LoggerManagerTreePtr getDefaultLogManager() {
+    return getTestLoggerManager()->getChild("ExecutorITF");
+  }
+
+  template <typename OtherResult>
+  ExecutorItf::RunResult makeRunResult(const OtherResult &other_result) {
```

Comment:
Please try to find a better name for `other_result`.
The meaning is not explained or clarified by the current name.

Probably the docstring for the method can help, but the name of the variable is still a potential subject of change.